src/components/CommentCard.jsx

```diff
@@ -0,0 +1,31 @@
+import React from "react";
+import { useEffect, useState } from "react";
+import { useParams } from "react-router-dom";
+import { getComments } from "../Api";
+
+export default function CommentCard() {
+  const { article_id } = useParams();
+  const [comments, setComments] = useState([]);
+
+  useEffect(() => {
+    getComments(article_id).then((response) => {
+      setComments(response.comments);
+    });
+  }, []);
```

Comment:
Don't forget to add _article_id_ to the useEffect's dependency array so that the fetch will re-run if the number in the URL gets changed